### DISCUSSION TITLE
Replay functionality doesn't work

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15437] [Q&A] [answered]
Watching a replay is no longer possible, the button is grayed out (isn't related to the imported replay, same thing across the game):

https://user-images.githubusercontent.com/53346335/139954514-dbf98f9d-d2d2-41b7-bf38-9f6718277201.mp4

[database.log](https://github.com/ppy/osu/files/7464114/database.log)
[runtime.log](https://github.com/ppy/osu/files/7464115/runtime.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15438
